src/msw/graphicsd2d.cpp

```diff
@@ -360,6 +360,221 @@ HRESULT WINAPI wxD3D11CreateDevice(
 }
 #endif
 
+#if wxUSE_PRIVATE_FONTS
+
+// This function is defined in src/msw/font.cpp.
+extern const wxArrayString& wxGetPrivateFontFileNames();
+
+namespace
+{
```

Comment:
Very minor, of course, but we typically don't indent the namespaces content, to avoid too deep indentation levels.